src/Host.tsx

```diff
@@ -44,8 +45,8 @@ export const Host = ({ children, style }: IHostProps): JSX.Element => {
     }
   }, []);
 
-  const mount = (children: React.ReactNode): number => {
-    const key = `portal_${Date.now()}_${Math.floor(Math.random() * 10000)}`;
+  const mount = (children: React.ReactNode): string => {
+    const key = uuidv4();
```

Comment:
For debugging purpose I think having portalize in the key would be useful.

```suggestion
    const key = `portalize_${uuidv4()}`;
```